python/orca/src/bigdl/orca/common.py

```diff
@@ -183,79 +180,95 @@ def init_orca_context(cluster_mode="local", cores=2, memory="2g", num_nodes=1,
            Default to be False and in this case the Ray cluster would be launched lazily when
            Ray is involved in Project Orca.
     :param kwargs: The extra keyword arguments used for creating SparkContext and
-           launching Ray if any.
-
+           launching Ray if any. 
+           For k8s-client mode, you should specify master as ${SPARK_MASTER} (k8s://https://localhost:8443)
+           and set container_image to ${K8s_SPARK_IMAGE}.
```

Comment:
Put this in cluster mode?
```
For k8s-client mode, you should additionally specify the arguments master and container_image.
```

Reply:
Better to keep this in client mode, I didn't see any arg is required in cluster mode(spark-submit).

Reply:
I mean cluster_mode argument....
A similar paragraph like `For "spark-submit", you are supposed to use spark-submit to submit the application.....` here https://github.com/intel-analytics/BigDL/pull/3397/files#diff-e52466efd112054f4c0e38a19903bc5e8e40a3b9caa3291c7b67110069b2d912R168

Reply:
> I mean cluster_mode argument.... A similar paragraph like `For "spark-submit", you are supposed to use spark-submit to submit the application.....` here https://github.com/intel-analytics/BigDL/pull/3397/files#diff-e52466efd112054f4c0e38a19903bc5e8e40a3b9caa3291c7b67110069b2d912R168

Haha sorry, that's a good idea.